src/main/java/com/cleanroommc/groovyscript/registry/ReloadableRegistryManager.java

```diff
@@ -68,6 +68,7 @@ public static <T> List<T> unmarkScripted(Class<?> registryClass, @SuppressWarnin
     @ApiStatus.Internal
     public static void init() {
         registryDummies.put(IRecipe.class, DummyRecipe::new);
+        new VanillaModule().init();
```

Comment:
what
Why not just `VanillaModule.loot.init()` :concern:

Reply:
Because I figured other VanillaModules added in the future may require some sort of setup, silly goose.

if you prefer it that way I can change it.

Reply:
At least make the method static...

Reply:
ah like these calls here right? 
![image](https://github.com/CleanroomMC/GroovyScript/assets/47100551/6beb0c52-e90f-4977-bd55-ad298135bf08)
I already reverted it to the way you'd initially suggested but if you'd like me to make VanillaModule.onReload() and afterScriptRun() static as well let me know. I can bundle that in with these changes.

Reply:
Wtf, i never noticed that